spaces/domains.json

```
@@ -135,4 +135,5 @@
   "gov.alpacafinance.org": "alpacafinance.eth",
   "dao.zfo.gg": "zfogg.eth",
   "vote.ensuser.com": "ensuser.eth"
+  "governance.diamondhandsfinance.org": "diamondhandsfinance.eth"
```

You miss a comma at the end of the line above your custom domain.
```suggestion
  "vote.ensuser.com": "ensuser.eth",
  "governance.diamondhandsfinance.org": "diamondhandsfinance.eth"
```